ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/DefaultBlockchain.java

```diff
@@ -408,6 +424,33 @@ private BlockAddedEvent handleChainReorg(
         currentNewChainWithReceipts.getBlock().getHash());
   }
 
+  private void maybeLogReorg(
+      final BlockWithReceipts newChainHeadWithReceipts,
+      final BlockWithReceipts oldChainWithReceipts,
+      final BlockWithReceipts commonAncestorWithReceipts) {
+    if (newChainHeadWithReceipts.getNumber() - commonAncestorWithReceipts.getNumber()
+            > reorgLoggingThreshold
+        || oldChainWithReceipts.getNumber() - commonAncestorWithReceipts.getNumber()
+            > reorgLoggingThreshold) {
+      LOG.warn(
+          "Chain Reorganization\n{}",
```

Comment:
Perhaps a block count?  
```
Chain Reorganization +4/-6 [... the rest...]
```
where +x is the distance from the ancestor on the new chan, -x is the distance of the old chain.

Reply:
Will do